src/tensorwaves/data/transform.py

```diff
@@ -1,14 +1,14 @@
-"""Implementations of `.DataConverter`."""
+"""Implementations of `.DataTransformer`."""
 
 from expertsystem.amplitude.kinematics import EventCollection, HelicityAdapter
 
-from tensorwaves.interfaces import DataConverter, DataSample
+from tensorwaves.interfaces import DataSample, DataTransformer
 
 
-class HelicityKinematicsConverter(DataConverter):
+class HelicityTransformer(DataTransformer):
     """Convert four-momentum tuples to variables in the helicity formalism.
 
-    Implementation of a `.DataConverter` based on the
```

Comment:
Shouldn't we also make the change `convert` -> `transform` in the interface definition?

Reply:
That's right. I was also wondering whether the naming in the `expertsystem` should be changed:
https://pwa.readthedocs.io/projects/expertsystem/en/0.7.0a0/api/expertsystem.amplitude.kinematics.html#expertsystem.amplitude.kinematics.HelicityAdapter.convert

Reply:
I would say so

Reply:
https://github.com/ComPWA/expertsystem/pull/524

Reply:
Ok good. Right now its a bit of a mixture anyways, since the `HelicityAdapater` in the `expertsystem` does not follow the same interface `EventCollection` vs `Dict[str, np.ndarray]`